### ISSUE TITLE
Philips Ambilight can't find the diyHue bridge

### ISSUE THREAD

**Issue by galaxxy22** [diyhue/diyHue#988]
### Issue does not already exist?

I have searched and found no existing issue

### Select Environment

Docker

### Home Assistant related?

No

### Description

I have tried installing diyHue via HomeAssistant and Docker. When I go to the settings of the TV and try and search for bridges, it only finds my original Philips Hue bridge, but not the diyHue bridge. The Hue app and Essentials app do add it succesfully.

There was actually one short time I did see it show up, but after adding it, it wouldn't work and was gone again.

Both the TV and diyHue are on the same network.

### Errorlog:

```
I don't know where to find the logs
```

### Steps to reproduce

- Go to the ambilight settings of the TV
- Search for available bridges

### Please enter your operating system details here

DiyHue latest version running on Docker running on a Linux server running on a VM in Virtualbox.

### What DiyHue version(branch) are you using?

master (latest)

**Comment by mariusmotea** [comment]
There are two methods to discover a Hue Bridge. Using local LAN services or using cloud based services. It is more easy to do it using cloud based services and because wich will not work with diyHue. Since your TV may have a successful discovery using cloud services (the original bridge) it will not fall back to local LAN discovery.

**Comment by galaxxy22** [comment]
> There are two methods to discover a Hue Bridge. Using local LAN services or using cloud based services. It is more easy to do it using cloud based services and because wich will not work with diyHue. Since your TV may have a successful discovery using cloud services (the original bridge) it will not fall back to local LAN discovery.

That makes sense. Any idea how to fix this? Disable the Philips Hue bridge temporairly?

**Comment by mariusmotea** [comment]
yes, but i think you need to keep it disconnected with more than 30 seconds before to perform a bridge discovery.

**Comment by mariusmotea** [comment]
Check by accessing https://discovery.meethue.com/ , it need to return `[]` when no bridges are registred to online discovery service.

**Comment by galaxxy22** [comment]
> Check by accessing https://discovery.meethue.com/ , it need to return `[]` when no bridges are registred to online discovery service.

This already returns [] for me without disabling the Hue bridge

**Comment by mariusmotea** [comment]
most likely you are not in the same network.

**Comment by galaxxy22** [comment]
I restarted the Hue bridge, the found it https://discovery.meethue.com/. Then disconnected again and managed to connect the TV to the diyHue bridge! Unfortunatly when I connect the Hue bridge again, after a few minutes the TV connected lights stop working, it was also quite laggy, but that might be because it is only a ESP8266

**Comment by mariusmotea** [comment]
ESP lights work better than original ones in entertainment. But if the tv is older it don't use the entertainment protocol, but regulat http api with 1 request/second.

**Comment by AidenShaw2020** [comment]
Hi @mariusmotea, i have similar problem as @galaxxy22.
diyHue is working OK for me on HUE app and HUE Essentials app, but not on TV.

On my first Philips TV with Android is hub correctly detected, bud on another one with Saphi OS wont.
Both TV´s are on same network,

I tried install diyHue/master in container as HOST and so as BRIDGE, and also directly on host, but its without change.

Can you help me please?
Thanks.

(For addition, other emulated services (for example: https://github.com/hass-emulated-hue/core) is detected OK.
